clvm-derive/src/to_clvm.rs

```diff
@@ -59,18 +236,17 @@ pub fn to_clvm(mut ast: DeriveInput) -> TokenStream {
 
     ast.generics
         .params
-        .push(GenericParam::Type(TypeParam::from(node_name.clone())));
+        .push(GenericParam::Type(node_name.clone().into()));
     let (impl_generics, _, _) = ast.generics.split_for_impl();
 
     quote! {
         #[automatically_derived]
-        impl #impl_generics #crate_name::ToClvm<#node_name> for #struct_name #ty_generics #where_clause {
+        impl #impl_generics #crate_name::ToClvm<#node_name> for #item_name #ty_generics #where_clause {
```

Comment:
"item" is a pretty generic name. Is there a more concrete name that can be used here? maybe "type" or "identifier". Either way, it could probably use a comment to describe what an "item" really is.